src/System.Data.SqlClient/tests/ExceptionTest.cs

```diff
@@ -0,0 +1,148 @@
+﻿using System.Collections;
+using Xunit;
+
+namespace System.Data.SqlClient.Tests
+{
+    public class ExceptionTest
+    {
+        // test connection string
+        string conn_string = "server=tcp:server,1432;database=test;uid=admin;pwd=SQLDB;connect timeout=60;";
+
+        // data value and server consts
+        private const string badServer = "NotAServer";
+        private const string sqlsvrBadConn = "A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)";
+        private const string execReaderFailedMessage = "ExecuteReader requires an open and available Connection. The connection's current state is closed.";
+        private const string orderIdQuery = "select orderid from orders where orderid < 10250";
```

Comment:
I've changed "string conn_string" to "private string connectionString" for styling convention.

The global / shared variable represent client error messages, and if any of these error messages were to change in the imported libraries, it makes it so much easier to update a single global variable than to search for a string in the entire file. If the use case changes for these exception tests, it would make sense to change the global variables to local variables at that point.